src/rpg_database/character_sheet/Focus.java

```diff
@@ -0,0 +1,57 @@
+package rpg_database.character_sheet;
+
+public class Focus {
+
+	private boolean isImproved;
+	private int value;
+	private Focuses focus;
+
+	public Focus(Focuses focus) {
+		this(focus, false);
+	}
+
+	public Focus(Focuses focus, boolean isFocusImproved) {
+		if (isFocusImproved) {
+			value = 3;
+			isImproved = true;
+		} else {
+			value = 2;
+			isImproved = false;
+		}
+		this.focus = focus;
+	}
+
+	public boolean isFocusImproved() {

```

Comment:
biztos hogy kell erre tartani egy isImproved tagváltozó? Miért nem elég visszaadni, hogy 
```java
return value > 2;
```
?

Reply:
Azt gondoltam jobban olvasható ha van egy isFocus mezőm is.

Reply:
De átírtam

Reply:
Jó elképzelés, de gondolj rá úgy, hogy ez esetben pontosan ez a függvény fogja betölteni a mező funkcionalitását, ezért nincs rá szükség.